packages/babel-preset-react-union/src/index.js

```diff
@@ -53,7 +49,7 @@ module.exports = () => ({
 				regenerator: true,
 			},
 		],
-		cjs && ['@babel/plugin-transform-modules-commonjs', { loose }],
+		'@babel/plugin-transform-modules-commonjs',
```

Comment:
I'm not exactly sure if this is correct, but it fixed a thing for me and did not break anything. I'll try and investigate.